test/libsolidity/gasTests/storage_costs.sol

```diff
@@ -0,0 +1,24 @@
+contract C {
```

Comment:
@hrkrshnn can we merge this test outside of this PR? So we can see the gas changes in this PR.

Reply:
The first commit actually shows the old costs. The new costs can be seen in the second commit.